test/e2e/Makefile

```diff
@@ -2,26 +2,34 @@
 
 BINARY       ?= kubernetes-on-aws-e2e
 VERSION      ?= $(shell git describe --tags --always --dirty)
-KUBE_VERSION ?= v1.14.6
```

Comment:
did we really forgot to update the e2e image to 1.14.8 before this?

Reply:
You're right but we only missed a few changes to the e2e tests: https://github.com/kubernetes/kubernetes/compare/v1.14.6...v1.14.8#diff-7cf22f9ddb0506fc122a9c459eb16f76